### DISCUSSION TITLE
error MSB6006: "yasm.exe" exited with code -1073741515.

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2404] [Q&A]
error MSB6006: "yasm.exe" exited with code -1073741515.
fresh windows 10 devkit install

**Comment by ghost** [comment]
see #2102 (install visual c++ 2010)

**Reply by winbamstudios** [reply]
i fixed it myself eventually

**Reply by Takkuxi** [reply]
How do you fix it please!! :(
